client/src/views/ProtubeScreen.vue

```diff
@@ -135,4 +203,26 @@ socket.on("new-video-timestamp", async (newStamp) => {
     if ((await player.getPlayerState()) === 2) player.playVideo();
   }
 });
+
+socket.on("queue-update", (newQueue) => {
+  queueDuration.value = newQueue.duration;
+  queue.value = newQueue.queue;
+});
+
+function updateCurrentlyPlaying() {
```

Comment:
This is for updating the progressbar but there is a simpler way of doing this by creating a ref variable and assigning that a percentage that updates on the "new-video-timestamp" socket